### PR TITLE
Support rendering Markdown

### PR DESCRIPTION
**Pull request by RileyYe**
## What I did

I utilized the library named `react-markdown` to support Markdown
rendering.

## Checklist

Hello! Welcome to send a PR to us! Here is a simple PR checklist:

- [x] Format your code, by `yarn format`. A good format is helpful for
      everyone.
- [ ] (NOT NOW) Make sure tests are passed.
- [ ] If you need to update the document, then update. The documents are here:
  - `README.md` in the root folder of project, about what the project is and
    why the project exists.
  - `README.md` in the package `back`, `front` and `theme-shortcut`(at
    the path `front/packages/theme-shortcut`).
  - In folder `docs`, there are the other documents.

### CODE REVIEW

File: front/src/page/News.tsx [hunk 1]
```diff
@@ -55,11 +57,7 @@ export default () => {
       >
         {new_.title}
       </Header>
-      <div
-        dangerouslySetInnerHTML={{
-          __html: replace(new_.content, /\n/g, '<br />'),
-        }}
-      />
+      <ReactMarkdown children={new_.content} remarkPlugins={[remarkGfm]} />
```
Comment (RileyYe) [inline front/src/page/News.tsx:60]: Here is the core code.

Comment (PeterlitsZo) [reply front/src/page/News.tsx:60]: Could you talk about the `remarkGfm` in your commit message? I am not sure what is it used for?

File: package.json [hunk 2]
```diff
@@ -13,5 +13,8 @@
     "prettier": "^2.5.1",
     "prettier-plugin-sql": "^0.4.1",
     "zx": "^5.2.0"
+  },
+  "dependencies": {
+    "yarn": "^1.22.18"
```
Comment (PeterlitsZo) [inline package.json:18]: yarn should be a global tool.

File: back/src/graphql/type/user.js [hunk 3]
```diff
@@ -159,4 +159,4 @@ export const signIn = {
     context.res.cookie('jwt', token, { httpOnly: true });
     return res;
   },
-};
\ No newline at end of file
+};
```
Comment (PeterlitsZo) [inline back/src/graphql/type/user.js:162]: You can put this change into another commit.

File: front/public/mockServiceWorker.js [hunk 4]
```diff
@@ -2,7 +2,7 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (0.36.7).
+ * Mock Service Worker (0.36.8).
```
Comment (PeterlitsZo) [inline front/public/mockServiceWorker.js:5]: You can put this change into another commit.

**Review by PeterlitsZo — CHANGES_REQUESTED** [review]

**Comment by PeterlitsZo** [comment]
1. Please make your git commit messages a fucking 80 characters long per line. 
2. Learn how to use the fucking git. I cannot help you every time.
3. Write code as much as you can.

**Comment by RileyYe** [comment]
？

**Comment by PeterlitsZo** [comment]
Thanks.
